contracts/CompoundHandler.sol

```diff
@@ -113,6 +121,14 @@ contract CompoundHandler is Handler, ReentrancyGuard {
             "deposit: Fail to supply to compound!"
         );
 
```

Comment:
Can define a common function, something like `transferCompOut()`,  to avoid duplicate code. Also, should we have a threshold when transfer out?